requirements.txt

```
@@ -37,6 +37,7 @@ matplotlib          # svg saver
 sh                  # ping
 psutil              # procmgr
 numpy               # npy pandas hdf5 arrow
+tomli               # toml (Python <3.11 only)
```

```suggestion
tomli; python_version < '3.11'               # toml
```

https://peps.python.org/pep-0508/#environment-markers

=)

---

(I could not really tell where to indent the `# toml` in the "suggestion" framework.

---

Thanks for catching this :+1: . It's a long line, so I just went with one space before the comment which seems to match up with other long lines (xlsb, importlib_metadata)